type/__clean_path/explorer/list

```diff
@@ -1,8 +1,8 @@
 #!/bin/sh -e
 #
-# 2019 Ander Punnar (ander-at-kvlt-dot-ee)
+# 2024 Ander Punnar (ander-at-kvlt-dot-ee)
 #
-# This file is part of cdist.
+# This file is part of skonfig.
 #
 # cdist is free software: you can redistribute it and/or modify
```

Comment:
`/^#/s/cdist/skonfig-base/g` ?

That's what I do when I touch a type.

Reply:
Done.

Reply:
Can you replace the other occurrences of cdist in the header, too?

Reply:
Please remind me what we decided about this:
```
ander@p14s:__clean_path$ grep -rF cdist .
./man.rst:cdist-type__clean_path(7)
./man.rst:cdist-type__clean_path - Remove files and directories in path recursively.
```

Reply:
I hope I don't lie but my memory says we decided to keep it for compatibility with cdist.
Since after all these types can be used with cdist as cdist types can be used with skonfig.
But I don't know if anybody is actually doing that.

However, this was a long time ago and since skonfig and cdist are now going into very different directions with us possibly considering to break compatibility with cdist types in the future, I'd be open to bring this topic back to table again.